app/components/user-avatar.js

```diff
@@ -19,7 +21,10 @@ export default Component.extend({
 	 * @returns {string}
 	 */
 	profileUrl: computed('profileName', function () {
+		const langPath = getLanguageCodeFromRequest(this.get('fastboot.request'));
+
 		return buildUrl({
+			langPath,
```

Comment:
langPath: getLanguageCodeFromRequest(this.get('fastboot.request'))

less code, same readability IMO